README.md

```diff
@@ -78,6 +82,20 @@ found in the `src/` directory.
     - stylesheets/
 ```
 
+`require('uswds')` will load all of Draft U.S. Web Design Standard's JavaScript onto the page. The uswds module itself does not export anything.
+
+The main Sass (SCSS) source file is here: 
+
+```
+node_modules/uswds/src/stylesheets/all.scss
+```
+
+The non-minified CSS that’s been precompiled is here:
```

Comment:
Sorry for the confusion, but yes. My note was for us to explicitly recommend the minified version when implementing via the downloaded version of the

@rogeruiz that is :100: okay in terms of using `require('uswds');` and having the non-minified version. Maybe something to add to our research to see what our users are wanting/needing for this method of implementation. //cc @carodew